Modelica/Blocks/Logical.mo

```diff
@@ -973,6 +973,41 @@ parameter \"terminationText\".
 
 </html>"));
   end TerminateSimulation;
+
+  block AssertCondition "Trigger error (or warning) and print message if assertion condition is not fulfilled"
+
+    Modelica.Blocks.Interfaces.BooleanOutput condition=true
+      "Trigger error (or warning) and print message when condition becomes false"
+      annotation (Dialog, Placement(transformation(extent={{200,-10},{220,10}})));
+    parameter String message="... assertion condition no longer true"
+      "Text that will be displayed when assertion condition is false";
+    parameter AssertionLevel assertionLevel=AssertionLevel.error
+      "Select warning (1) or error (2)";
+  equation
+    assert(condition,message,assertionLevel);
+    annotation (Icon(coordinateSystem(
+          preserveAspectRatio=true,
+          extent={{-200,-20},{200,20}},
+          initialScale=0.2), graphics={
+          Rectangle(
+            fillColor={235,235,235},
+            fillPattern=FillPattern.Solid,
+            lineThickness=5,
+            borderPattern=BorderPattern.Raised,
+            extent={{-200,-20},{200,20}}),
+          Text(extent={{-166,-15},{194,15}}, textString="%condition"),
+          Rectangle(
+            fillColor={0,140,72},
+            fillPattern=FillPattern.Solid,
+            borderPattern=BorderPattern.Raised,
+            extent={{-194,-14},{-168,14}}),
+          Text(
+            textColor={0,0,255},
+            extent={{-200,22},{200,46}},
+            textString="%name")}), Documentation(info="<html>
+<p>In the parameter menu, a <strong>time varying</strong> expression can be defined via variable <strong>condition</strong>, for example &quot;condition = x &lt; 0&quot;, where &quot;x&quot; is a variable that is declared in the model in which the &quot;AssertCondition&quot; block is present. If this expression becomes <strong>false</strong>, the simulation will either abort with a message indicating the cause of the error (if assertion level is set to error) or not abort with a message indicating the cause of the warning (if assertion level is set to warning).</p>
```

Comment:
Can we avoid speaking of the _parameter menu_ for a more tool independent documentation?

Reply:
Any suggestions? This wording was pulled from the **TerminateSimulation** block.

Reply:
Maybe it can just be omitted:
```suggestion
<p>The block is used by specifying a (possibly) <strong>time varying</strong> expression for the variable <strong>condition</strong>, for example &quot;condition = x &lt; 0&quot;, where &quot;x&quot; is a variable that is declared in the model in which the &quot;AssertCondition&quot; block is present. If this expression becomes <strong>false</strong>, the simulation will either abort with a message indicating the cause of the error (if assertion level is set to error) or not abort with a message indicating the cause of the warning (if assertion level is set to warning).</p>
```